Source/DynamicPartHider/HelperFuncs.cs

```diff
@@ -0,0 +1,73 @@
+using System.Linq;
+using UnityEngine;
+
+namespace RealismOverhaul
+{
+    public static class HelperFuncs
+    {
+        public static bool IsPartAvailable(AvailablePart ap)
+        {
+            if (ap == null)
+            {
+                return false;
+            }
+            var specLevel = HighLogic.CurrentGame.Parameters.CustomParams<RealismOverhaulSettings>().speculativeLevel;
```

Comment:
var is bad because the type isn't apparent from the right side of the assignment